pkg/oss/utils.go

```diff
@@ -63,6 +65,15 @@ func GetMetaDataAsync(resource string) string {
 	}
 }
 
+// GetRegionID Get RegionID from Metadata first, then Environment Variables
+func GetRegionID() string {
+	regionID := GetMetaDataAsync(regionTag)
```

Comment:
Metadata may returns 500 / 403 error code with non empty response body.

Reply:
but i guess http.Get will return error in GetMetaData func